app/controllers/supplementary/SummaryPageController.scala

```diff
@@ -73,7 +73,13 @@ class SummaryPageController @Inject()(
               .saveSubmissionResponse(submission)
               .flatMap { _ =>
                 exportsMetrics.incrementCounter(submissionMetric)
-                customsCacheService.remove(supplementaryCacheId).map(_ => Ok("Supplementary Declaration submitted"))
+                customsCacheService
+                  .remove(supplementaryCacheId)
+                  .map(
+                    _ =>
+                      Redirect(controllers.supplementary.routes.ConfirmationPageController.displayPage())
+                        .flashing(prepareFlashScope(lrn.getOrElse(""), conversationId))
```

Comment:
I'm not sure about `.getOrElse`... I'm thinking about some exception when lrn is empty. As I know this field cannot be empty.